### PR TITLE
Whatsnew fix

### PR DESCRIPTION
**Pull request by bdenham**
## Purpose of this pull request

This pull request (PR) adds the Cloud Docker 1.2.1 release to the whats new page.

## Affected DevDocs pages

https://devdocs.magento.com/whats-new.html

image

### CODE REVIEW

File: src/_data/whats-new.yml [hunk 1]
```diff
@@ -14,6 +14,13 @@ entries:
   link: https://github.com/magento/devdocs/pull/8488
   contributor: jeff-matthews
   profile: https://github.com/jeff-matthews
+- description: Published release notes for [Cloud Docker 1.2.1 release notes](https://devdocs.magento.com/cloud/release-notes/mcd-release-notes.html).
```
Comment (shrielenee) [inline src/_data/whats-new.yml:17]:
```suggestion
- description: Published release notes for [Cloud Docker 1.2.1 release notes](https://devdocs.magento.com/cloud/release-notes/mcd-release-notes.html#v121).
```

Comment (shrielenee) [inline src/_data/whats-new.yml:17]:
```suggestion
- description: Published release notes for the [Cloud Docker 1.2.1 release notes](https://devdocs.magento.com/cloud/release-notes/mcd-release-notes.html).
```

**Review by shrielenee — APPROVED** [review]
Approved with revision

**Review by dshevtsov — APPROVED** [review]

**Comment by bdenham** [comment]
running tests

**Pull request merged**